File: FundamentalsJava/src/NegatiationEx/Vacation.java
package NegatiationEx;

import java.util.Scanner;

public class Vacation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int people = Integer.parseInt(scanner.nextLine());
        String typePeople = scanner.nextLine();
        String typeOfDay = scanner.nextLine();
        double price = 0;
        if (typePeople.equals("Business") && people >= 100){
            people = people - 10;
        }
        if (typeOfDay.equals("Friday")){
            if (typePeople.equals("Students")){
                price = 8.45 * people;
            }else if (typePeople.equals("Business")){
                price = 10.90 * people;
            }else if (typePeople.equals("Regular")){
                price = 15 * people;
            }
        }else if (typeOfDay.equals("Saturday")){
            if (typePeople.equals("Students")){
                price = 9.80 * people;
            }else if (typePeople.equals("Business")){
                price = 15.60 * people;
            }else if (typePeople.equals("Regular")){
                price = 20 * people;
            }
        }else if (typeOfDay.equals("Sunday")){
            if (typePeople.equals("Students")){
                price = 10.46 * people;
            }else if (typePeople.equals("Business")){
                price = 16 * people;
            }else if (typePeople.equals("Regular")){
                price = 22.50 * people;
            }
        }
        if (typePeople.equals("Students") && people >= 30){
            price = price - (price * 0.15);
        }else if(typePeople.equals("Regular") && people >= 10 && people <= 20){
            price = price - (price * 0.05);
        }
        System.out.printf("Total price: %.2f", price);
    }
}

File: JsFront-end/jsSyntaxFundExercise/vacation.js
function totalPrice(peopleCount, typeGroup, week) {
  let totalPrice;
  switch (typeGroup) {
    case "Students":
      if (week === "Friday") {
        totalPrice = 8.45 * peopleCount;
      } else if (week === "Saturday") {
        totalPrice = 9.8 * peopleCount;
      } else if (week === "Sunday") {
        totalPrice = 10.46 * peopleCount;
      }
      if (peopleCount >= 30) {
        totalPrice -= totalPrice * 0.15;
      }
      break;
    case "Business":
      if (peopleCount >= 100) {
        peopleCount -= 10;
      }
      if (week === "Friday") {
        totalPrice = 10.9 * peopleCount;
      } else if (week === "Saturday") {
        totalPrice = 15.6 * peopleCount;
      } else if (week === "Sunday") {
        totalPrice = 16 * peopleCount;
      }
      break;
    case "Regular":
      if (week === "Friday") {
        totalPrice = 15 * peopleCount;
      } else if (week === "Saturday") {
        totalPrice = 20 * peopleCount;
      } else if (week === "Sunday") {
        totalPrice = 22.5 * peopleCount;
      }
      if (peopleCount >= 10 && peopleCount <= 20) {
        totalPrice -= peopleCount * 0.05;
      }
      break;
  }
  console.log(`Total price: ${totalPrice.toFixed(2)}`);
}
totalPrice(170, "Business", "Friday");
